cmd/metal-api/internal/service/switch-service.go

```diff
@@ -628,8 +626,12 @@ func makeSwitchResponse(s *metal.Switch, ds *datastore.RethinkStore) (*v1.Switch
 
 	nics := makeSwitchNics(s, ips, machines)
 	cons := makeSwitchCons(s)
+	ss, err := ds.GetSwitchStatus(s.ID)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Should go into `findSwitchReferencedEntites`, which is where the other references entities are gathered.

Reply:
done